Automattic/WP-Job-Manager
# Drop IE11 support from scripts build

Fixes #2834 

### Changes Proposed in this Pull Request

* Remove custom list of browser targets and use the WordPress default
* This effectively removes IE11 from the supported browsers

### Testing Instructions

* Build the scripts with `npm run build:assets`
* Use WordPress 6.6 on a test site
* Visit the job listing admin page, and check that there are no console errors
* Enable stats
* Visit a job listing, and check that there are no console errors and the stat request is sent


### Release Notes

* Remove support for Internet Explorer 11
* Fix Wordpress 6.6 compatibility 

### New or Updated Hooks and Templates





----

| Plugin build for 4809257d5eedbc3cde7b86147794652684901f39 <a href="#"><img width=600></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2024/07/wp-job-manager-zip-2835-4809257d.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2024/07/2835-4809257d)             |
